assets/styles/index.scss

```diff
@@ -213,28 +222,46 @@ body {
   justify-content: space-around;
   top: calc(calc(100%) - 150px);
   
-
+  
+  
   .color-codes {
-    width: calc(calc(100%)/5.3);
+    height: calc(calc(12vw)/5);
+    width: calc(calc(500%)/7);
     display: flex;
     justify-content: center;
     align-content: center;
     
-
+    
     .hex-code {
       background: none;
+      opacity: 60%;
+      border-radius: 3px;
       text-align: center;
       border: none;
       height: calc(calc(12vw)/5);
       width: calc(calc(500%)/7);
-      color: rgba(255, 255, 255, 0.767);
```

Comment:
ben de mukemmel degilmisim, `0.767` ne amk : D , `0.75` in neyi eksik ya da `0.8`  hahha sanki cok fark var